drought/southern-africa/drought_impact_2023.Rmd

```diff
@@ -0,0 +1,90 @@
+---
+title: "Untitled"
+author: "Pauline"
+date: "`r Sys.Date()`"
+output: html_document
+runtime: shiny
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+
+library(ripc)
+library(tidyverse)
+library(countrycode)
+library(sf)
+library(raster)
+library(sp)
+library(gghdx)
+
+load_source_sans_3()
+options(scipen = 999)
+sf_use_s2(FALSE)
+
+ipc_country_data <- ipc_get_country()
+
+esa_iso2 <- c("AO", "SZ", "LS", "MG", "MW", "MZ", "NA", "MW", "MZ", "ZA", "ZM", "ZW")
```

Comment:
Botswana? I find it strange that Botswana is missing in white, while others included but no IPC data are dark grey. Maybe put all countries with no IPC data as white?